samcli/lib/build/app_builder.py

```diff
@@ -216,20 +216,27 @@ def update_template(template_dict: Dict, original_template_path: str, built_arti
 
             resource_type = resource.get("Type")
             properties = resource.setdefault("Properties", {})
+            package_type = properties.get("PackageType", ZIP)
```

Comment:
Let's move this back to be inline. I understand this is duplicated throughout but package_type is not on all resources we update here. The way this is written suggests it could also be applied to Layers (which is does not).

Reply:
We should maybe move the layers code before this line then:
```python
    if resource_type in [SamBaseProvider.SERVERLESS_LAYER, SamBaseProvider.LAMBDA_LAYER]:
        properties["ContentUri"] = store_path
        continue
```
it's weird that it is in the middle of the others